kubernetes/linux/setup.sh

```diff
@@ -84,6 +84,14 @@ echo "$(fluent-bit --version)" >> packages_version.txt
 # install fluentd using the mariner package
 # sudo tdnf install rubygem-fluentd-1.14.6 -y
 gem install fluentd -v "1.14.6" --no-document
+
+# remove the certs from the gem
+cd /usr/lib/ruby/gems/3.1.0/gems/fluentd-1.14.6/test/plugin_helper/data/cert
```

Comment:
1. can we purge entire test directory ?
2. can we parameterized the fluentd version so that we dont need to update this for every fluentd  version?

Reply:
Updated the PR with these changes